.github/workflows/Dockerfile.yml

```
@@ -39,12 +40,12 @@ jobs:
     strategy:
       matrix:
         network: [testnet, pubnet, standalone]
-        options: ["", --enable-horizon-captive-core]
+        options: ["", "--enable-horizon-captive-core --enable-soroban-rpc"]
```

@sreuland To address the issue you're experiencing you need to retain the `--enable-horizon-captive-core` as its own option. The options as input to the strategy matrix.
```suggestion
        options: ["", --enable-horizon-captive-core, "--enable-horizon-captive-core --enable-soroban-rpc"]
```

---

ah, thanks, done here - https://github.com/stellar/quickstart/pull/367/commits/8019a99efc33bdb864f8bf78b9d8d8a94d8af74e